RELEASE/scripts/autoscend.ash

```diff
@@ -10425,10 +10419,23 @@ boolean LX_craftAcquireItems()
 		}
 	}
 
-	#Can we have some other way to check that we have AT skills?
-	if((item_amount($item[Antique Accordion]) == 0) && (item_amount($item[Aerogel Accordion]) == 0) && isUnclePAvailable() && (my_meat() > 12500) && (have_skill($skill[The Ode to Booze])) && (my_class() != $class[Accordion Thief]) && (auto_my_path() != "G-Lover"))
+	#Can we have some other way to check that we have AT skills? Checking all skills just to be sure.
+	if((item_amount($item[Antique Accordion]) == 0) && (item_amount($item[Aerogel Accordion]) == 0) && isUnclePAvailable() && (my_meat() >= npc_price($item[Antique Accordion])) && (my_class() != $class[Accordion Thief]) && (auto_my_path() != "G-Lover"))
 	{
-		buyUpTo(1, $item[Antique Accordion]);
+		boolean buyAntiqueAccordion = false;
+		boolean[skill] songs = $skills[Inigo\'s Incantation of Inspiration, The Ballad of Richie Thingfinder, Chorale of Companionship, The Ode to Booze, Ur-Kel\'s Aria of Annoyance, Carlweather\'s Cantata of Confrontation, The Sonata of Sneakiness, Paul\'s Passionate Pop Song, Aloysius\' Antiphon of Aptitude, Fat Leon\'s Phat Loot Lyric, The Polka of Plenty, Donho\'s Bubbly Ballad, Prelude of Precision, Elron\'s Explosive Etude, Benetton\'s Medley of Diversity, Dirge of Dreadfulness, Stevedave\'s Shanty of Superiority, The Psalm of Pointiness, Brawnee\'s Anthem of Absorption, Jackasses\' Symphony of Destruction, The Power Ballad of the Arrowsmith, Cletus\'s Canticle of Celerity, Cringle\'s Curative Carol, The Magical Mojomuscular Melody, The Moxious Madrigal];
```

Comment:
i see we have another complete list of AT songs in auto_util.ash, probably worth making an accessor method that returns a static list and updating your new code and `shrugAT` to use it (and double check there isnt anywhere else using it):

```
boolean[effect] accordionThiefSongs(){
  static boolean[skill] SONGS = $skills[Inigo\'s Incantation of Inspiration, The Ballad of Richie Thingfinder, Chorale of Companionship, The Ode to Booze, Ur-Kel\'s Aria of Annoyance, Carlweather\'s Cantata of Confrontation, The Sonata of Sneakiness, Paul\'s Passionate Pop Song, Aloysius\' Antiphon of Aptitude, Fat Leon\'s Phat Loot Lyric, The Polka of Plenty, Donho\'s Bubbly Ballad, Prelude of Precision, Elron\'s Explosive Etude, Benetton\'s Medley of Diversity, Dirge of Dreadfulness, Stevedave\'s Shanty of Superiority, The Psalm of Pointiness, Brawnee\'s Anthem of Absorption, Jackasses\' Symphony of Destruction, The Power Ballad of the Arrowsmith, Cletus\'s Canticle of Celerity, Cringle\'s Curative Carol, The Magical Mojomuscular Melody, The Moxious Madrigal];
  return SONGS;
}
```

Reply:
The ones in shrugAT() are **effects** set in a particular order from Most to Least important to the script in order to determine which we should shrug when trying to apply a new buff.

The ones in this are **skills** set in no particular order.

Reply:
well effects and skills are interchangeable in this context (just call to_skill/to_effect as needed). And since only one cares about order they could both use the same list. Though Ill grant you it the ranked order is unintuitive for re use purposes. A comment might suffice. We now just have two essentially identical lists to maintain in different places. Ill leave it up to you.

Reply:
I added comments.

When I was tinkering with this the addition of "The " in the skill names caused lines to spam in mafia every time we tested.

Reply:
Done! Thanks for the help!